src/Bicep.Core.Samples/Files/baselines/InvalidExpressions_LF/Completions/symbols.json

```diff
@@ -17,6 +21,10 @@
     "label": "and",
     "kind": "variable",
     "detail": "and",
+    "documentation": {
```

Comment:
I assume this causes no issues?

Reply:
Consider: There would be a lot fewer changes if you didn't write this out for empty string.